config.yml

```diff
@@ -125,7 +125,7 @@ tron:
 
 # [VET] VeChain: https://www.vechain.org
 vechain:
- api: https://explore.veforge.com/api
+  api: https://vethor-pubnode.digonchain.com
```

Comment:
Why do we call it `api` if in reality we call `rpc` ?. I understand meaning for explorers as 3rd party solution.

Reply:
this is an RPC node API